tests/workers/network/smoke.sh

```diff
@@ -18,9 +19,15 @@ rm -f /tmp/tcpaccept.log
 # in case if it's still running from a previous run
 pkill berserker || true
 
+# make berserkers verbose
+#export RUST_LOG=trace
+
+# start the server before bpftrace, to skip first accept
+echo "Starting the server..."
+berserker tests/workers/network/workload.server.toml &> /tmp/server.log &
```

Comment:
[nit] Launching in background ('&') does not guarantee the server will have completed initialization before bpftrace is started. This would require that berserker decides when to daemonize to give back control.

Reply:
Good point, I'll add some mechanisms to make sure it's fully initialized.

Reply:
And it's there.

Reply:
This is not what I had imagined, but it is simple and should work. Thanks !